WordPress/Classes/ViewRelated/Comments/Controllers/CommentsViewController.m

```diff
@@ -235,7 +241,27 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     if (self.isSidebarModeEnabled) {
         self.commentDetailViewController.isSidebarModeEnabled = true;
         self.commentDetailViewController.navigationItem.hidesBackButton = YES;
-        [self showDetailViewController:self.commentDetailViewController sender:nil];
+
+        if (@available(iOS 26.0, *)) {
+            __weak __typeof(self) weakSelf = self;
+            self.commentDetailViewController.closeBarButtonItem = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemClose primaryAction:[UIAction actionWithHandler:^(__kindof UIAction * _Nonnull _) {
+                [weakSelf.currentSplitViewController hideColumn:UISplitViewControllerColumnInspector];
```

Comment:
Need to call `setViewController:nil forColumn: .inspector` here too?

Reply:
Done. I also rebased.